templates/tsconfig.json.tmpl

```diff
@@ -1,7 +1,8 @@
 {
 	"extends": "../../tsconfig",
 	"compilerOptions": {
-		"outDir": "dist"
+		"outDir": "dist",
+		"esModuleInterop": true
```

Comment:
Needed after adding `@ethereumjs/` packages

Reply:
Please make this only for the package where necessary required, not for templates. This change applies to all packages.

Reply:
There appears to be a symbolic link between this file in the template and all packages - assumed this was intentional, but maybe it wasn't?